### PR TITLE
Swipe touch  - hammerjs replaced with contact-js 

### PR DESCRIPTION
**Pull request by arumuganainar73**
1. Removed hammerjs package from package json
2. Replaced touch actions with contact js (https://biodiv.github.io/contactjs/)

### CODE REVIEW

File: libs/ngu/carousel/src/lib/ngu-carousel/ngu-carousel.component.ts [hunk 1]
```diff
@@ -339,89 +336,79 @@ export class NguCarousel<T>
...
-        });
+    const options = {
+      "supportedGestures": [Pan]
+    };
 
-        if (this.vertical.enabled) {
-          this._nguCarouselHammerManager.on(hammer, 'panup').subscribe((ev: any) => {
-            this._touchHandling('panleft', ev);
-          });
+    this._pointerListener = new PointerListener(this._touchContainer.nativeElement, options);
```
Comment (santoshyadavdev) [inline libs/ngu/carousel/src/lib/ngu-carousel/ngu-carousel.component.ts:348]: Just to understand if this property is used to hold the reference, should er also unsubscribe to this listener onDestroy

**Review by santoshyadavdev — CHANGES_REQUESTED** [review]

**Comment by santoshyadavdev** [comment]
can you add contactjs here https://github.com/uiuniversal/ngu-carousel/blob/master/libs/ngu/carousel/package.json#L8 and remove hammer, also remove all the reference to hammerjs and replace with contactjs

File: libs/ngu/carousel/src/lib/ngu-carousel/ngu-carousel.component.ts [hunk 2]
```diff
@@ -339,89 +336,79 @@ export class NguCarousel<T>
   }
 
   ngOnDestroy() {
-    this._hammer?.destroy();
     this._destroy$.next();
   }
 
   /** Get Touch input */
   private _setupHammer(): void {
```
[the remark this answers is not in the document]
Comment (arumuganainar73) [reply libs/ngu/carousel/src/lib/ngu-carousel/ngu-carousel.component.ts]: Done

**Comment by arumuganainar73** [comment]
> can you add contactjs here https://github.com/uiuniversal/ngu-carousel/blob/master/libs/ngu/carousel/package.json#L8 and remove hammer, also remove all the reference to hammerjs and replace with contactjs

done

File: libs/ngu/carousel/src/lib/ngu-carousel/ngu-carousel.component.ts [hunk 3]
```diff
@@ -153,6 +149,8 @@ export class NguCarousel<T>
 
   private _destroy$ = new Subject<void>();
 
+  private _pointerListener: any;
```
[the remark this answers is not in the document]
Comment (arumuganainar73) [reply libs/ngu/carousel/src/lib/ngu-carousel/ngu-carousel.component.ts]: No required, Removed

File: libs/ngu/carousel/src/lib/ngu-carousel/ngu-carousel.component.ts (hunk 1 above)
Comment (arumuganainar73) [reply libs/ngu/carousel/src/lib/ngu-carousel/ngu-carousel.component.ts]: No required, Removed

**Comment by santoshyadavdev** [comment]
can you resolve the conflict please

**Pull request closed without merging**